<ui version="4.0" >
 <class>BulmaContBase</class>
 <widget class="QMainWindow" name="BulmaContBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>BulmaCont - Contabilidad GPL</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/iconbulmacont.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVentana" >
    <property name="title" >
     <string>&amp;Ver</string>
    </property>
    <addaction name="actionPlan_Contable" />
    <addaction name="actionAsientos" />
    <addaction name="actionApuntes_Contables" />
    <addaction name="actionLibro_Mayor" />
    <addaction name="actionLibro_Diario" />
    <addaction name="actionBalance_de_Cuentas" />
   </widget>
   <widget class="QMenu" name="menuAsiento" >
    <property name="title" >
     <string>&amp;Asientos</string>
    </property>
    <addaction name="actionAbrir_Asientos" />
    <addaction name="actionOrdenar_Asientos" />
    <addaction name="separator" />
    <addaction name="actionAsiento_de_Apertura" />
    <addaction name="actionAsiento_de_Regularizacion" />
    <addaction name="actionAsiento_de_Cierre" />
   </widget>
   <widget class="QMenu" name="menuVer" >
    <property name="title" >
     <string>Ventana</string>
    </property>
    <addaction name="actionPantalla_Completa" />
    <addaction name="actionOrdenar_Ventanas" />
    <addaction name="actionOrganizaci_n_en_Cascada" />
    <addaction name="actionIndexador" />
   </widget>
   <widget class="QMenu" name="menuAyuda" >
    <property name="title" >
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAyuda" />
    <addaction name="actionAcerca_de" />
   </widget>
   <widget class="QMenu" name="menuMaestro" >
    <property name="title" >
     <string>&amp;Maestro</string>
    </property>
    <addaction name="actionFormas_de_Pago" />
    <addaction name="actionTipos_de_IVA" />
    <addaction name="separator" />
    <addaction name="actionCentros_de_Coste" />
    <addaction name="actionCanales" />
    <addaction name="separator" />
    <addaction name="actionPaises" />
   </widget>
   <widget class="QMenu" name="menuEmpresa" >
    <property name="title" >
     <string>&amp;Empresa</string>
    </property>
    <addaction name="actionConfiguracion" />
    <addaction name="actionSalir" />
   </widget>
   <widget class="QMenu" name="menuListados" >
    <property name="title" >
     <string>&amp;Listados</string>
    </property>
    <addaction name="actionCuentas_Anuales" />
    <addaction name="separator" />
    <addaction name="separator" />
    <addaction name="actionListado_347" />
    <addaction name="separator" />
    <addaction name="actionAmortizaciones" />
   </widget>
   <widget class="QMenu" name="menuHerramientas" >
    <property name="title" >
     <string>&amp;Herramientas</string>
    </property>
    <addaction name="actionRecalcular_Saldos_Iniciales" />
    <addaction name="actionCerrar_Asientos" />
    <addaction name="actionSustituir_Cuentas" />
    <addaction name="actionBloquear_Fechas" />
    <addaction name="separator" />
    <addaction name="actionCanal_por_Defecto" />
    <addaction name="actionCentro_de_Coste_por_Defecto" />
   </widget>
   <addaction name="menuEmpresa" />
   <addaction name="menuAsiento" />
   <addaction name="menuListados" />
   <addaction name="menuVentana" />
   <addaction name="menuMaestro" />
   <addaction name="menuHerramientas" />
   <addaction name="menuVer" />
   <addaction name="menuAyuda" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionConfiguracion" />
   <addaction name="actionPlan_Contable" />
   <addaction name="actionApuntes_Contables" />
   <addaction name="actionAsientos" />
   <addaction name="actionLibro_Mayor" />
   <addaction name="actionLibro_Diario" />
   <addaction name="actionBalance_de_Cuentas" />
   <addaction name="actionInicio" />
   <addaction name="actionAnterior" />
   <addaction name="actionSiguiente" />
   <addaction name="actionFin" />
  </widget>
  <action name="actionConfiguracion" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_configuracion.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Propiedades de la empresa...</string>
   </property>
   <property name="iconText" >
    <string>Configuracion</string>
   </property>
   <property name="toolTip" >
    <string>Configuracion</string>
   </property>
  </action>
  <action name="actionSalir" >
   <property name="text" >
    <string>&amp;Salir del programa</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbrir_Asientos" >
   <property name="text" >
    <string>&amp;Espaciar asientos</string>
   </property>
  </action>
  <action name="actionOrdenar_Asientos" >
   <property name="text" >
    <string>&amp;Ordenar asientos</string>
   </property>
  </action>
  <action name="actionAsiento_de_Apertura" >
   <property name="text" >
    <string>Asiento de a&amp;pertura</string>
   </property>
  </action>
  <action name="actionAsiento_de_Regularizacion" >
   <property name="text" >
    <string>Asiento de &amp;regularizacion</string>
   </property>
  </action>
  <action name="actionAsiento_de_Cierre" >
   <property name="text" >
    <string>Asiento de &amp;cierre</string>
   </property>
  </action>
  <action name="actionCuentas_Anuales" >
   <property name="text" >
    <string>Cuentas a&amp;nuales</string>
   </property>
  </action>
  <action name="actionMasas_Patrimoniales" >
   <property name="text" >
    <string>Masas &amp;patrimoniales</string>
   </property>
  </action>
  <action name="actionAmortizaciones" >
   <property name="text" >
    <string>&amp;Amortizaciones</string>
   </property>
  </action>
  <action name="actionPlan_Contable" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_cuentas.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Plan Contable</string>
   </property>
  </action>
  <action name="actionAsientos" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_asiento.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Asientos</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionApuntes_Contables" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_lapiz.xpm</iconset>
   </property>
   <property name="text" >
    <string>Apuntes &amp;contables</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionLibro_Mayor" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_mayor.xpm</iconset>
   </property>
   <property name="text" >
    <string>Libro &amp;Mayor</string>
   </property>
  </action>
  <action name="actionLibro_Diario" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_diario.xpm</iconset>
   </property>
   <property name="text" >
    <string>Libro &amp;Diario</string>
   </property>
  </action>
  <action name="actionBalance_de_Cuentas" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_balance.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Balance de Cuentas</string>
   </property>
  </action>
  <action name="actionFormas_de_Pago" >
   <property name="text" >
    <string>Formas de &amp;pago</string>
   </property>
  </action>
  <action name="actionTipos_de_IVA" >
   <property name="text" >
    <string>Tipos de &amp;IVA</string>
   </property>
  </action>
  <action name="actionCentros_de_Coste" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_centroCoste.xpm</iconset>
   </property>
   <property name="text" >
    <string>Centros de cos&amp;te</string>
   </property>
  </action>
  <action name="actionCanales" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_canales.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Canales</string>
   </property>
  </action>
  <action name="actionAsientos_Inteligentes" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_asientoListo.xpm</iconset>
   </property>
   <property name="text" >
    <string>Asientos Inteligentes</string>
   </property>
  </action>
  <action name="actionImprimir" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_filePrint.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Imprimir</string>
   </property>
  </action>
  <action name="actionGuardar" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_save.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Guardar</string>
   </property>
  </action>
  <action name="actionActualizar" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_reload.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Actualizar</string>
   </property>
  </action>
  <action name="actionFiltrar" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_find.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Filtrar</string>
   </property>
  </action>
  <action name="actionRecalcular_Saldos_Iniciales" >
   <property name="text" >
    <string>&amp;Recalcular saldos iniciales</string>
   </property>
  </action>
  <action name="actionSustituir_Cuentas" >
   <property name="text" >
    <string>&amp;Sustituir cuentas</string>
   </property>
  </action>
  <action name="actionBloquear_Fechas" >
   <property name="text" >
    <string>&amp;Bloquear fechas</string>
   </property>
  </action>
  <action name="actionCanal_por_Defecto" >
   <property name="text" >
    <string>&amp;Canal por defecto</string>
   </property>
  </action>
  <action name="actionCentro_de_Coste_por_Defecto" >
   <property name="text" >
    <string>Centro de cos&amp;te por defecto</string>
   </property>
  </action>
  <action name="actionPantalla_Completa" >
   <property name="text" >
    <string>&amp;Pantalla completa</string>
   </property>
   <property name="shortcut" >
    <string>F11</string>
   </property>
  </action>
  <action name="actionOrdenar_Ventanas" >
   <property name="text" >
    <string>Ordenar &amp;ventanas</string>
   </property>
  </action>
  <action name="actionOrganizaci_n_en_Cascada" >
   <property name="text" >
    <string>Organización en &amp;cascada</string>
   </property>
  </action>
  <action name="actionAyuda" >
   <property name="text" >
    <string>A&amp;yuda</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="actionAcerca_de" >
   <property name="text" >
    <string>&amp;Acerca de BulmaCont</string>
   </property>
  </action>
  <action name="actionSiguiente" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_forward.xpm</iconset>
   </property>
   <property name="text" >
    <string>Siguiente</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionAnterior" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_back.xpm</iconset>
   </property>
   <property name="text" >
    <string>Anterior</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionInicio" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_start.png</iconset>
   </property>
   <property name="text" >
    <string>Inicio</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="actionFin" >
   <property name="icon" >
    <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_finish.png</iconset>
   </property>
   <property name="text" >
    <string>Fin</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+End</string>
   </property>
  </action>
  <action name="actionIndexador" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Indexador</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionListado_347" >
   <property name="text" >
    <string>Listado &amp;347</string>
   </property>
  </action>
  <action name="action" >
   <property name="text" >
    <string>"</string>
   </property>
  </action>
  <action name="action_2" >
   <property name="text" >
    <string>''</string>
   </property>
  </action>
  <action name="actionPaises" >
   <property name="text" >
    <string>Paises</string>
   </property>
  </action>
  <action name="actionCerrar_Asientos" >
   <property name="text" >
    <string>Cerrar Asientos</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>activated()</signal>
   <receiver>BulmaContBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
